Update signing-in.rst (#4311)
mattermost/docs: source/help/getting-started/signing-in.rst

> Signing in

Reviewer: ```suggestion
Signing In
```

==========

> Accessing your Mattermost workspace

Reviewer: ```suggestion
Accessing your Mattermost Workspace
```

------------------------------------

> An ivite is required to access Mattermost. A member of your Mattermost workspace can send an invitation through an email or by providing you a link. 

Reviewer: ```suggestion
An invite is required to access Mattermost. A member of your Mattermost workspace can send an invitation through an email or by providing you a link.
```

Reviewer 2: I committed this suggestion in error.

Sign In 
--------

> To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your administrator or in an invitation email. We recommend bookmarking the site so you can easily find it again. 

Reviewer: ```suggestion

To sign in, navigate to the Mattermost sign-in page. You can get the URL of the sign-in page from your System Admin or in an invitation email. We recommend bookmarking the site so you can easily find it again.
```

> After signing in, the team that appears first on your team sidebar will open.  If you have not joined a team, the Team Selection

Reviewer 2: ```suggestion
After signing in, the team that appears first on your team sidebar will open. If you have not joined a team, the Team Selection
```

Switching Teams
---------------

You can switch between teams you have joined using the team sidebar that appears to the left of your channel list on the left-hand sidebar.

> .. image:: ../../images/team-sidebar.png

Reviewer 3: @cwarnermm separate from this PR these images should be updated

Reviewer 2: Thanks, @esethna. I've logged a docs ticket (https://mattermost.atlassian.net/browse/MM-33036) for this update.

Setting Your Status
-------------------

You can set your status as **Online**, **Away**, **Do not disturb**, or **Offline** by choosing a status from the menu that appears when you select your avatar at the top of the channel list. **Do not disturb** disables desktop, email, and push notifications.

> .. image:: ../../images/avatar-online-status-218x247.png

Reviewer 3: @cwarnermm separate from this PR these images should be updated
